CMakeLists.txt

```diff
@@ -176,7 +176,9 @@ elseif ( NOT MAKE_DOCS_ONLY )
     HPCC_ADD_SUBDIRECTORY (services "PLATFORM")
     HPCC_ADD_SUBDIRECTORY (thorlcr "PLATFORM")
     HPCC_ADD_SUBDIRECTORY (testing)
-    HPCC_ADD_SUBDIRECTORY (configuration)
+    if ( NOT WIN32 )
+        HPCC_ADD_SUBDIRECTORY (configuration)
```

Comment:
I assume config manager is not meant to be supported on windows.  If it is, then more work is needed.